File: src/echo/EchoClient.java
package echo;

import org.apache.thrift.TException;
import org.apache.thrift.protocol.TBinaryProtocol;
import org.apache.thrift.protocol.TProtocol;
import org.apache.thrift.transport.TSocket;
import org.apache.thrift.transport.TTransport;
import org.apache.thrift.transport.TTransportException;

public class EchoClient {
    private static final int PORT = 9000;

    public static void main(String[] args) {
            try {
                    TTransport transport = new TSocket("localhost", PORT);
                    TProtocol protocol = new TBinaryProtocol(transport);
					EchoService.Client client = new EchoService.Client(protocol);
                    transport.open();
                    transport.close(); 
            } catch (TTransportException e) {
                    e.printStackTrace();
            } catch (TException e) {
                    e.printStackTrace();
            }
    }
}

File: WebContent/Web/app/lib/echo_service/echo_client.js
var EchoClient = (function() {
    return {
    	echo : function(message) {
    	    var transport = new Thrift.Transport("http://localhost:8080/EchoService/EchoServlet");
    	    var protocol  = new Thrift.Protocol(transport);
    	    var client    = new EchoServiceClient(protocol);

    	    try {
    	      result = client.echo(message);
    	      return result;
    	    } catch(ouch){
    	      //have to handle error
    	    }
    	}
    };
}());